docs/src/main/sphinx/connector/hive.md

```diff
@@ -341,14 +341,19 @@ Hive connector documentation.
   - `false`
 :::
 
-## Storage
+### File system access configuration
 
-The Hive connector supports the following storage options:
+The connector supports native, high-performance file system access to object
+storage systems:
 
-- {doc}`Amazon S3 <hive-s3>`
```

Comment:
Why do we delete the links here, but leave them in the Delta Lake docs? Also delete from Delta Lake?

Reply:
Yes.. also deleted there now. The Hive connector still has more mess .. but I will follow up on that.